js/models/messages.js

```diff
@@ -962,6 +963,17 @@
       };
     },
 
+    copyPubKey() {
+      if (this.isIncoming()) {
+        clipboard.writeText(this.get('source'));
+      } else {
+        clipboard.writeText(this.OUR_NUMBER);
```

Comment:
What would the `source` be for a non-incoming message?
Asking to see if it's worth differentiating incoming/outgoing or no...

Reply:
It was undefined and was causing writeText to error
Come to think of it maybe I should double check the source is defined anyway